example/src/main/resources/MessageFilterImpl.java

```diff
@@ -1,39 +0,0 @@
-/**
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.rocketmq.example.filter;
-
-import org.apache.rocketmq.common.filter.FilterContext;
-import org.apache.rocketmq.common.filter.MessageFilter;
-import org.apache.rocketmq.common.message.MessageExt;
-
-public class MessageFilterImpl implements MessageFilter {
```

Comment:
the MessageFilter interface could also be removed.

Reply:
Could you please elaborate

Reply:
Because org.apache.rocketmq.common.filter.MessageFilter is not used anywhere exept here, we have org.apache.rocketmq.store.MessageFilter, so org.apache.rocketmq.common.filter.MessageFilter can be deleted.

Reply:
oh.. okay, should I do that in a different pr as the tittle for this one say's Removing MessageFilterImpl , I think i should what do you think?

Reply:
Ok, you can remove it later